src/Renderer/Base.php

```diff
@@ -0,0 +1,628 @@
+<?php
+/**
+ * Base response class for Joomla Webservices. Based on \JDocument in the CMS
+ *
+ * @package    Webservices
+ * @copyright  Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Webservices\Renderer;
+
+use Joomla\Webservices\Application;
+
+use Joomla\DI\Container;
+use Joomla\DI\ContainerAwareTrait;
+use Joomla\DI\ContainerAwareInterface;
+
+/**
+ * Document class, provides an easy interface to parse and display a document
+ *
+ * @since  __DELPOY_VERSION__
```

Comment:
It's not a typo - it's just the standard we happen to use in the framework e.g. https://github.com/joomla-framework/application/commit/185b68fe55537281385b637a90e11856ea01c8da